docs.md

```diff
@@ -243,7 +243,7 @@ def beSmallerThan(value: Int) = ValidationRule[Int](s"be smaller than $value") {
   
 val child: RequestReader[User] = 
   (RequiredParam("name") ~
-  RequiredIntParam("age").should(beNonNegative and beSmallerThan(18))) map {
+  RequiredParam("age").as[Int].should(beNonNegative and beSmallerThan(18))) map {
```

Comment:
I think we should change `beSmallerThan` to `beLessThan` or `beLessThanOrEqualTo` so that we're consistent with the operator names these functions are representing.